vendor/daterangepicker.js

```diff
@@ -1,5 +1,5 @@
 /**
-* @version: 2.1.30
+* @version: 2.1.27

```

Comment:
why do we need downgraded this version?

Reply:
we fixed the bug in 2.1.27 version during the development time.